src/runner/browser-job.ts

```diff
@@ -205,26 +211,58 @@ export default class BrowserJob extends AsyncEventEmitter {
         return true;
     }
 
+    private _getTestControllerQueue (connectionId: string): TestRunController[] {
+        if (this._noConcurrencyQueue[connectionId]?.length)
+            return this._noConcurrencyQueue[connectionId];
+
+        return this._testRunControllerQueue;
+    }
+
+    private _updateTestControllerQueues (controller: TestRunController, connectionId: string): void {
+        const { test }                   = controller;
+        const { noConcurrency, fixture } = test;
```

Comment:
Let's escape from `const { noConcurrency, fixture } = test;` destructurization.  My reason for this suggestion:
1. You use these properties only once
2. It will be clearer to understand where exactly you get these properties in place and where you use them
3. `const { test } = controller;` assigning will look prettier